x-pack/platform/plugins/shared/fleet/server/services/epm/packages/cache.ts

```diff
@@ -17,12 +17,15 @@ const cacheStore = new AsyncLocalStorage<CacheSession>();
 
 const PACKAGE_INFO_CACHE_SIZE = 20;
 const PACKAGE_ASSETS_MAP_CACHE_SIZE = 1;
+const AGENT_TEMPLATE_ASSETS_MAP_CACHE_SIZE = 5;
```

Comment:
We can cache more than package assets map as it only contains agent template and manifest, and not all ES|kibana assets like `PACKAGE_ASSETS_MAP_CACHE_SIZE`